src/components/Landing/Landing.js

```diff
@@ -39,6 +39,10 @@ const COURSE_DATA_MAPPING = {
   'CSS Layouts': {
     icon: 'info',
     subTitle: 'Learn about CSS fundamentals on layouts.',
+  },
+  'SOLID Principles': {
+    icon: 'solid',
+    subTitle: 'Learn the SOLID Design Principles and how to apply them.',
```

Comment:
I might add that these are SOLID Architecture principles; on a CSS heavy platform, people might see "design" and think something else.